docs/sources/setup/install/helm/reference.md

```diff
@@ -2149,7 +2151,12 @@ null
     "s3": null,
     "s3ForcePathStyle": false,
     "secretAccessKey": null,
-    "signatureVersion": null
+    "signatureVersion": null,
```

Comment:
Can you run `make -C docs sources/setup/install/helm/reference.md` as mentioned in https://drone.grafana.net/grafana/loki/26714/5/2?